content/en/blog/_posts/2024-04-01-validating-admission-policy-ga/index.md

```diff
@@ -0,0 +1,216 @@
+------
+layout: blog
+title: "Kubernetes 1.30: Validating Admission Policy Is Generally Available"
+slug: validating-admission-policy-ga
+date: 2024-04-01
+canonicalUrl: https://www.k8s.dev/blog/2024/04/01/validating-admission-policy/
+---
+
+**Author:** Jiahui Feng (Google)
+
+We are excited to announce that Validating Admission Policy has reached its General Availability
+as part of Kubernetes 1.30 release. If you have not yet read about this new declarative alternative to
+validating admission webhooks, it may be interesting to read our
+[previous post](/blog/2022/12/20/validating-admission-policies-alpha/) about the new feature.
```

Comment:
What do you think of:
```suggestion
validating admission webhooks, it may be interesting to read the
[previous post](/blog/2022/12/20/validating-admission-policies-alpha/) by my two of my colleagues
at Google, introducing
this new API.
```
?

Reply:
Looks good. Thank you.